assets/js/src/core/modules/element/dynamic-types/defintinitions/objects/data-related/types/abstract/dynamic-type-object-data-abstract-input.tsx

```diff
@@ -33,4 +33,18 @@ export abstract class DynamicTypeObjectDataAbstractInput extends DynamicTypeObje
       />
     )
   }
+
+  getVersionObjectDataComponent (props: InputProps): React.ReactElement<AbstractObjectDataDefinition> {
```

Comment:
Why do we need to override this method here? The abstract-input extends the regular input. Shouldn't be then our default implmentation from the `DynamicTypeObjectDataAbstract` be enough?